board.py

```diff
@@ -59,6 +61,21 @@ def output(self, n):
                 str += self.row_string(row)
                 row.clear()
         return str
-
-    def __str__(self):
-        return "temporary"
+    
+    def render(self, Bool, skip, guess_value, board, game_over):
+        # decoder
+        if (Bool or skip) and not game_over:
+            print(board.output(2))
+            if guess_value == 1:
+                print("\nCorrect!\tPress \"Enter\" to continue: ")
+            if guess_value == 2:
+                print("\nThat is the other team's word! Your turn is over.\tPress \"Enter\" to continue: ")
+            if guess_value == 3:
+                print("\nThat is a nuetral word. Your turn is over.\tPress \"Enter\" to continue: ")
+        # codegiver
+        else:
+            if not game_over:
+                system("clear")
+                input("\nIt is now the Codegiver turn.\nWARNING: all word guessers must look away from the screen!\n\nPress \"Enter\" when you are ready: ")
+                system("clear")
+            print(board.output(1))
```

Comment:
I feel like I'm possibly overlapping the process input and the render with the Codegiver warning requiring user input. I felt that this was the best way to do it, but let me know if I should be printing my warning in a different way.